lib/routes/home/widgets/payments_list/payment_item_amount.dart

```diff
@@ -34,7 +34,7 @@ class PaymentItemAmount extends StatelessWidget {
         builder: (context, currencyState) {
           final fee = _paymentInfo.feeMsat ~/ 1000;
           final amount = currencyState.bitcoinCurrency.format(
-            _paymentInfo.amountMsat ~/ 1000,
+            (_paymentInfo.amountMsat ~/ 1000).abs(),
```

Comment:
@ademar111190 so it seems that we decided to use absolute values for PaymentType.Received & PaymentType.Sent.  Perhaps the same should be for PaymenetType.ClosedChannel? Meaning we should fix the amount in the sdk to be absolute?

Reply:
@roeierez I believe it is a better approach to always use positive numbers because financial transactions always have a negative flow for one side and a positive flow to another side, so how to display or how to calculate should be the responsibility of the code dealing with such numbers.

e.g. which looks better?

```
balance = balance + feePaid;
```
or 
```
balance = balance - feePaid;
```

The second is better to understand when you are reading the code IMHO.

@erdemyerebasmaz It has been a while since I tested this flow on breezmobile, but the last time I tested it (when I did move the texts to the translations files) it was working as expected.

Reply:
@ademar111190 exactly, this is why I thought the `_paymentInfo.amountMsat` should come positive from the SDK. Isn't it aligned with what you wrote above?

Reply:
Yes it is, I'll close this PR to make this change on the SDK